packages/@aws-cdk/aws-backup/lib/vault.ts

```diff
@@ -108,27 +121,71 @@ export enum BackupVaultEvents {
   BACKUP_PLAN_MODIFIED = 'BACKUP_PLAN_MODIFIED',
 }
 
+abstract class BackupVaultBase extends Resource implements IBackupVault {
+  // The name of a logical container where backups are stored.
+  public abstract readonly backupVaultName: string;
+  // The ARN of the backup vault.
+  public abstract readonly backupVaultArn: string;
+
+  /**
+   * Grant the actions defined in actions to the given grantee
+   * on this Backup Vault resource.
+   *
+   * @param grantee Principal to grant right to
+   * @param actions The actions to grant
+   */
+  public grant(grantee: iam.IGrantable, ...actions: string[]): iam.Grant {
```

Comment:
according to the [AWS docs](https://docs.aws.amazon.com/aws-backup/latest/devguide/creating-a-vault-access-policy.html), wildcard actions (`"*"`) are not allowed; please add a validation